docs/conftest.py

```diff
@@ -0,0 +1,25 @@
+#
+#  Copyright (C) 2022
+#  MIT
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+import pytest
+
+@pytest.fixture(autouse=True)
+def add_sherpa_test_data_dir(doctest_namespace):
+    doctest_namespace["data_dir"] = 'sherpa-test-data/sherpatest/'
```

Comment:
So, we have `sherpa.utils.testing.get_datadir` which could be used here. Or perhaps either of the `test_data_path` or `make_data_path` decorators from `sherpa/conftest.py` (although are they in scope here???). Using these fixtures could be good, since it lets you know that the data directory is required (assuming it all hooks up nicely).

Reply:
`conftest.py` is only looked for the the current tree, not in a sibling tree - there is a specific warning in the pytest docs for cases like ours where docs and code are in separate trees. We *could* move the pytest doc into the base dir, but I don't think it belongs there. That's where this is parallel (i.e. a copy of) whats in the `sherpa/conftest.py`. At the minimum, I'll add a comment here to explain that. I did not know about (or forgot) `sherpa.utils.testing.get_datadir`, so I'll look if I can use that.